src/index.js

```diff
@@ -225,6 +217,22 @@ function getConfig(filePath, eslintPath) {
   }
 }
 
+async function resolvePrettierOptions(
+  filePath,
+  prettierPath,
+  prettierOptions = {},
+) {
+  /*
+   * The order of precedence (lowest to highest) is:
+   * - Options from a config file
+   * - prettierOptions
+   */
+
+  const prettier = getPrettier(prettierPath)
+  const optionsFromFile = await prettier.resolveConfig(filePath)
```

Comment:
Does this actually return the options from the file? I'm not sure it did last I looked at `resolveConfig` but maybe it was updated?